### ISSUE TITLE
iOS 11.4 Beetle PSX no longer working 

### ISSUE THREAD

**Issue by jgriggles** [libretro/beetle-psx-libretro#345]
I really have no idea what’s wrong and little to no knowledge of what I’m doing but while I was underway and unable to change or update anything, Beetle PSX stopped loading any of my roms/ISO’s. PSX reARMed works on my iPhone but I do not know how to transfer any of my save files or save states ( though I read somewhere you can’t copy save states to different emulators.) since it’s stopped working I’ve reloaded RetroArch, and updated everything I possible could through the RetroArch GUI but to no avail. Any help I can recieve on getting beetle up and running or how to transfer my save files to PSX reARMed, so I don’t lose all the time I’ve put into some of these games, would be greatly appreciated. Thank you.

**Comment by hizzlekizzle** [comment]
Is it crashing? Can you get a log?

**Comment by jgriggles** [comment]
Yes it crashes RetroArch and force closes the app after about 15 seconds of it trying to load. And how do I get a log? Again sorry for my incompetence and thank you for your help!

**Comment by hizzlekizzle** [comment]
I think you have to run it with debugging enable in xcode...? I'm not altogether sure about that, sorry.
